jujugui/static/gui/src/app/state/test-state.js

```diff
@@ -1018,7 +1022,7 @@ describe('State', () => {
 
       const tests = ROOT_RESERVED.map(root => {
         return {
-          path: `http://abc.com:123/${root}/u/lukewh/cheese`,
```

Comment:
This URL is also invalid on anything but `new`

Do we care to fail in that case? Maybe not...